### PR TITLE
Bump kube 1.15

### PR DESCRIPTION
**Pull request by jberkhahn**

### CODE REVIEW

File: vendor/github.com/pmorie/go-open-service-broker-client/v2/fake/fake.go [hunk 1]
```diff
@@ -282,10 +288,13 @@ type ProvisionReaction struct {
 	Error    error
 }
 
-func (r *ProvisionReaction) react(_ *v2.ProvisionRequest) (*v2.ProvisionResponse, error) {
+func (r *ProvisionReaction) react(req *v2.ProvisionRequest) (*v2.ProvisionResponse, error) {
 	if r == nil {
 		return nil, UnexpectedActionError()
 	}
+	if req.ServiceID == "" || req.PlanID == "" || req.OrganizationGUID == "" || req.SpaceGUID == "" {
```
Comment (MHBauer) [inline vendor/github.com/pmorie/go-open-service-broker-client/v2/fake/fake.go]: New validation was added. Need to make sure all the fakes have these fields.

File: Gopkg.lock [hunk 2]
```diff
@@ -620,15 +641,15 @@
...
+  digest = "1:687d8e855c34a3b334efbd49c64eae500500bc2701ab16d286990f2634595fa5"
   name = "github.com/pmorie/go-open-service-broker-client"
   packages = [
     "v2",
     "v2/fake",
     "v2/generator",
   ]
   pruneopts = "NUT"
-  revision = "6988c0983446576f2cefc90112028a66e6137233"
+  revision = "a894d21a6d93b2b01b434b2226d806cc1240f79d"
```
Comment (MHBauer) [inline Gopkg.lock]: shouldn't be updating this anyway.

I can see that the project in Toml is dependent on 'master'. It should be locked down, and versioned.

**Comment by MHBauer** [comment]
Do the update of the library in a separate PR.
Lock it to some specific version/revision rather than the master branch.

**Comment by MHBauer** [comment]
forgotten comment, based on discussion on Friday

The fake Namespace test-namespace is being added, multiple times, to the fakeKubeClinet fake, and I'm not sure where it's coming from.

**Comment by jberkhahn** [comment]
had to force push up to add a real commit message. this should be good to merge now

**Comment by mszostok** [comment]
/lgtm

but still with the same comment: https://github.com/kubernetes-sigs/service-catalog/pull/2683#pullrequestreview-273836746

**Pull request merged**